### PR TITLE
v0.3.0

### PR DESCRIPTION
**Pull request by speediedan**
Checklist
* [X] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [X] Reset the build number to `0` (if the version changed)
* [X] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [X] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by speediedan** [comment]
Note: the initial finetuning-scheduler 0.3.x release on [conda-forge](https://github.com/conda-forge/finetuning-scheduler-feedstock/pull/12/) is currently pending and will be released once the upstream [
pytorch-lightning conda-forge package](https://anaconda.org/conda-forge/pytorch-lightning/files) is available. Installation of FTS via pip within a conda env is the recommended installation approach (both in the interim and in general actually)
This PR has been superseded by #14
